docs/docs/overview/quickstart/get-started-with-vanilla-js.md

```diff
@@ -0,0 +1,315 @@
+    ></notification-center-component>
+
+    <script>
+      // here you can attach any callbacks, interact with the web component API
+      // You can also move this to a seperate js file and have all the callbacks and other logic there
+      let nc = document.getElementsByTagName('notification-center-component')[0];
+      nc.onLoad = () => console.log('hello world!');
+    </script>
+  </body>
+</html>
+```
+
+You can add all the logic you want to add to the `script` element at the bottom or move it all into a seperate JavaScript file (don't forger to link to the JavaScript file in this case).
+
+Here's an example of such an HTML file:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://novu-web-component.netlify.app/index.js"></script>
+
+    <title>Vanilla JS app</title>
+  </head>
+
+  <body>
+    <div>
+      <h1>Notification generator</h1>
+      <notification-center-component
+        style="display: inline-flex"
+        application-identifier="<YOUR_NOVU_APPLICATION_IDENTIFIER>"
+        subscriber-id="123"
+      ></notification-center-component>
+    </div>
+    <div class="">
+      <form action="#" method="post" class="form">
+        <input class="input" placeholder="Enter notification text" type="text" name="description" />
+        <button class="btn">Send</button>
+      </form>
+    </div>
+  </body>
+  <script src="./app.js"></script>
+</html>
+```
+
+And the corresponding script file is as follows:
+
+```javascript
+const btn = document.querySelector('.btn');
+const input = document.querySelector('.input');
+const form = document.querySelector('.form');
+
+const onClickHandler = async (e) => {
+  e.preventDefault();
+  const desc = input.value;
+  console.log(desc);
+  try {
+    const resp = await fetch('<The API route from your backend>', {
+      method: 'POST',
+      body: JSON.stringify({
+        description: desc,
+      }),
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    input.value = '';
+    console.log(resp);
+  } catch (err) {
+    console.log(err);
+  }
+};
+
+form.addEventListener('submit', onClickHandler);
+
+// here you can attach any callbacks, interact with the web component API
+let nc = document.getElementsByTagName('notification-center-component')[0];
+nc.onLoad = () => console.log('hello world!');
+```
+
+The front-end as well as the back-end code for the sample application can be found [here](https://github.com/novuhq/vanillajs-quickstart) for illustration.
+
+## Code to trigger the notification
+
+We can trigger a notification by simply executing the trigger code snippet we get from the Workflow tab in the Novu web dashboard as shown in the image below:
+
+![Code snippet to trigger a notification](https://res.cloudinary.com/dxc6bnman/image/upload/v1686865543/guides/Screenshot_2023-06-16_at_3.15.35_AM_wqjvzh.png)
+
```

Comment:
We should also add the code snippet here for people that just want to copy straight from this guide.

Reply:
Done ✅